### PR TITLE
Pytorch version is updated to 1.13.1

### PR DESCRIPTION
**Pull request by aleksandr-mokrov**
Pytorch version is updated to 1.13.1. Experiments were tested with python3.10

### CODE REVIEW

**Review by psfoley — APPROVED** [review]

**Comment by itrushkin** [comment]
Should we also bump versions being installed in CI workflows? For example, in `.github/workflows/interactive-kvasir.yml`.

File: .github/workflows/fets-challenge.yml [hunk 1]
```diff
@@ -36,7 +36,7 @@ jobs:
       run: |
         cd fets_challenge
         pwd
-        pip install torch==1.8.2+cpu torchvision==0.9.2+cpu torchaudio===0.8.2 -f https://download.pytorch.org/whl/lts/1.8/torch_lts.html
+        pip install torch==1.13.1+cpu torchvision==0.14.1+cpu torchaudio===0.13.1 -f https://download.pytorch.org/whl/lts/1.8/torch_lts.html
```
Comment (psfoley) [inline .github/workflows/fets-challenge.yml]: The GaNDLF Package has a hard dependency on 1.8.2. This should be reverted.

Comment (psfoley) [reply .github/workflows/fets-challenge.yml]: https://github.com/mlcommons/GaNDLF/blob/0.0.14/setup.py#L88

**Comment by aleksandr-mokrov** [comment]
> Should we also bump versions being installed in CI workflows? For example, in .github/workflows/interactive-kvasir.yml.
> The GaNDLF Package has a hard dependency on 1.8.2. This should be reverted.
@itrushkin, @psfoley all versions are bumped, excluding fets-challenge.yml that uses GaNDLF Package.

**Pull request merged**